admin/common_auth/forms.py

```diff
@@ -0,0 +1,5 @@
+from django import forms
+
+class LoginForm(forms.Form):
+    username = forms.CharField(label=(u'User Name'), required=True)
```

Comment:
@cwisecarver I had noticed this and should have looked into it deeper - I can just change the naming here. But after I get the post variable, and call authenticate, I still need to say 
authenticate(username=email, password=password)  
because MyUser uses email as username_field, right? 
i.e. it wouldn't be authenticate(email=email, password=password)